docker/build_artifacts/sagemaker/python_service.py

```diff
@@ -257,8 +258,17 @@ def validate_model_dir(self, model_path):
             for dirname in dirs:
                 if dirname.isdigit():
                     versions.append(dirname)
-        if versions:
-            return True
+        return self.validate_model_versions(versions)
+
+    def validate_model_versions(self, versions):
+        if not versions:
+            return False
+        for v in versions:
+            if v.isdigit():
+                # TensorFlow model server will succeed with any versions found
```

Comment:
[Minor] If you have a link to the documentation stating that, it would be nice to include in the comment.

Reply:
Out of curiosity, the version is a single digit? Not a number or a major/minor version pattern string?
Can we add a test for the above, if appropriate? (I see your "invalid_version" test, but I'm specifically referring to the combinations above. Documentation could also clear up my confusion)

Reply:
Yes, the version is int64 according to their source code: https://github.com/tensorflow/serving/blob/master/tensorflow_serving/sources/storage_path/static_storage_path_source.proto#L9

For python, "00000123".isdigit() returns True.

For TensorFlow model server succeed with at least one valid version found, it was tested with this case: https://github.com/aws/sagemaker-tensorflow-serving-container/tree/master/test/resources/mme/half_plus_three